client/src/pages/contribute.js

```diff
@@ -10,10 +10,12 @@ import {
 } from 'grommet'
 import { Button } from 'components/Button'
 import { ContributeDownloadPDFButton } from 'components/ContributeDownloadPDFButton'
+import { ContributePhaseOutBanner } from 'components/ContributePhaseOutBanner'
```

Comment:
```suggestion
import { ContributeClosedCard } from 'components/ContributeClosedCard'
```

Since this isn't technically a banner can we rename this.